### ISSUE TITLE
Broken Google maps

### ISSUE THREAD

**Issue by simonihmig** [emberjs/website#3749]
## Current Behavior

Google Map cannot be loaded. See below for a screenshot.

## Expected Behavior

Google Maps should load correctly.

## Steps to Reproduce

1. visit https://www.emberjs.com/community/meetups/

## Possible Solution

Seems GMaps is loaded without an API key, which does not work anymore.

## Sources

![image](https://user-images.githubusercontent.com/1325249/50449745-7c959e00-0929-11e9-8744-ef672862ff92.png)

**Comment by mansona** [comment]
Howdy folks 👋 

I've just implemented leaflet in the new website here https://github.com/ember-learn/ember-website/pull/96/ but I don't know if we need to fix it on the old one before the new one is launched 🤔 

@MelSumner any thoughts? do you think we can hold off and just release the new website to fix this?

**Comment by MinThaMie** [comment]
I think just the API key they recently introduced is missing: https://developers.google.com/maps/documentation/javascript/get-api-key

**Comment by simonihmig** [comment]
> I think just the API key they recently introduced is missing

Yes, but that requires providing billing details, and costs could occur depending on the actual usage volume. So that's probably a blocker for an OSS project without a corporate backer willing to pay that!?

**Comment by MinThaMie** [comment]
True :( It's sad that they choose to do this...

**Comment by mansona** [comment]
I wouldn't worry about this issue too much as it is solved on the new website: https://ember-beta-website.netlify.com/community/meetups

I don't feel comfortable closing this issue until the swap-over has actually happened though 🤔

**Comment by simonihmig** [comment]
@mansona nice! 

Had to file an issue though: https://github.com/ember-learn/ember-website/issues/152 🤷‍♂️
